### ISSUE TITLE
Language not supported

### ISSUE THREAD

**Issue by mathieudebrito** [Triple-T/gradle-play-publisher#33]
Hi there, 

I translated my descriptions and all in many languages, 
Now, when I want to upload the translations, I got this error : 
> com.google.api.client.googleapis.json.GoogleJsonResponseException: 400 Bad Request
  {
    "code" : 400,
    "errors" : [ {
      "domain" : "androidpublisher",
      "message" : "Language not supported.",
      "reason" : "unsupportedListingLanguage"
    } ],
    "message" : "Language not supported."
  }

Is that possible to get more info, like which language is concerned ?

Thank you, this plugin is just awesome ;)

**Comment by bhurling** [comment]
Please try running the publish task with the `--debug` switch. You will get some outputs similar to those lines:

```
20:47:21.961 [INFO] [com.google.api.client.http.HttpTransport] -------------- REQUEST  --------------
PUT https://www.googleapis.com/androidpublisher/v2/applications/{package}/edits/14426522780959551550/listings/{locale}
Accept-Encoding: gzip
Authorization: <Not Logged>
User-Agent: gradle-play-publisher Google-HTTP-Java-Client/1.19.0 (gzip)
Content-Type: application/json; charset=UTF-8
Content-Length: 647
```

You have to watch out for the last occurence of that ` ---- REQUEST ---- ` line. Below that line you should find a URL that ends in a locale.

**Comment by mathieudebrito** [comment]
Ok thanks, I will try it :)

**Comment by mathieudebrito** [comment]
When I add `--debug`to the script parameters, I see nothing more. Did I miss something ?

**Comment by bhurling** [comment]
Hi, I don't think it's a good idea to test the locale before uploading. Google is constantly adding support for new locales and I don't really want to keep track of that. But perhaps we can enrich their error message so the user does not have to parse through the debug log to see what's going on

**Comment by mathieudebrito** [comment]
You got a point about Google adding new locales...
Enriching the error message seems to be a better idea indeed

**Comment by bhurling** [comment]
Fixed in 1.0.1

**Comment by mathieudebrito** [comment]
:+1:  thx
